packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -37,6 +37,8 @@ const DynamicKeyStream = BasicStream.extend({
     let key = this.keyDep.getValue();
     if (typeof key === 'string') {
       return key;
+    } else if (typeof key === 'number') {
+      return '' + key;
```

Comment:
I added a coercion in both `ember-glimmer` and `ember-htmlbars` because of the assertion here: https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/property_get.js#L49

Perhaps there's a better place to add this logic?